blocks/video/src/components/media.tsx

```diff
@@ -242,35 +242,54 @@ export const Media: FunctionComponent<
   };
 
   return (
-    <div ref={blockRef}>
-      {draftSrc ? (
-        <MediaWithCaption
-          src={draftSrc}
-          caption={draftCaption}
-          onCaptionChange={(caption) => setDraftCaption(caption)}
-          onCaptionConfirm={() => updateData({ src: draftSrc })}
-          onReset={resetComponent}
-          type="video"
-          readonly={readonly}
-        />
-      ) : (
-        <>
-          {errorString && (
-            <ErrorAlert
-              error={errorString}
-              onClearError={() => setErrorString(null)}
-            />
-          )}
-          <UploadMediaForm
-            onUrlConfirm={onUrlConfirm}
-            onFileChoose={(file) => handleImageUpload({ file })}
-            onUrlChange={(nextDraftUrl) => setDraftUrl(nextDraftUrl)}
-            loading={loading}
+    <>
+      <div ref={blockRef}>
+        {draftSrc ? (
+          <MediaWithCaption
+            src={draftSrc}
+            caption={draftCaption}
+            onCaptionChange={(caption) => setDraftCaption(caption)}
+            onCaptionConfirm={() => updateData({ src: draftSrc })}
+            onReset={resetComponent}
             type="video"
             readonly={readonly}
           />
-        </>
-      )}
-    </div>
+        ) : (
+          <>
+            {errorString && (
+              <ErrorAlert
+                error={errorString}
+                onClearError={() => setErrorString(null)}
+              />
+            )}
+            <UploadMediaForm
+              onUrlConfirm={onUrlConfirm}
+              onFileChoose={(file) => handleImageUpload({ file })}
+              onUrlChange={(nextDraftUrl) => setDraftUrl(nextDraftUrl)}
+              loading={loading}
+              type="video"
+              readonly={readonly}
+            />
+          </>
+        )}
+      </div>
+      {draftSrc ? (
+        <script
+          type="application/ld+json"
+          /* eslint-disable-next-line react/no-danger */
+          dangerouslySetInnerHTML={{
+            // @note – using JSON.stringify to prevent potential XSS
+            __html: JSON.stringify({
+              "@context": "https://schema.org/",
+              "@type": "VideoObject",
```

Comment:
sorry to drip-feed comments in, but can we add a note / `@todo` calling out the fact that this is missing [required properties](https://developers.google.com/search/docs/appearance/structured-data/image-license-metadata#structured-data-type-definitions)?

Reply:
Done